### ISSUE TITLE
How to calculate the P(object) and P(class/object) for training images ?

### ISSUE THREAD

**Issue by shamanez** [gliese581gg/YOLO_tensorflow#31]
I know this implementation is only for testing. But I went through the paper got to know the loss function and all. But what I could't understand is how to obtain above values for training images ?  There is no clear explanation for that in their paper.
